components/gitpod-db/src/typeorm/entity/db-team-settings.ts

```diff
@@ -30,6 +30,9 @@ export class DBOrgSettings implements OrganizationSettings {
     @Column("json", { nullable: true })
     restrictedEditorNames?: string[] | null;
 
+    @Column("varchar", { nullable: true })
```

Comment:
I'd really really love us to not use null, as I never saw it working as expected. :confused: 
E.g. this is what we get back from the DB layer when [storing an empty shape](https://github.com/gitpod-io/gitpod/tree/gpl/se/collaborator):
```
      + expected - actual

      -{
      -  "defaultRole": null,
      -  "defaultWorkspaceImage": null,
      -  "allowedWorkspaceClasses": null,
      -  "pinnedEditorVersions": null,
      -  "restrictedEditorNames": null,
      -  "workspaceSharingDisabled": false
      -}
      +{}
  ```

Given that we use the same pattern already I'd say it's outside the scope of this PR, though... :cry: